.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -17,7 +17,8 @@ Learn more about contributing: [CONTRIBUTING.md](https://github.com/nf-core/blob
 - [ ] If you've fixed a bug or added code that should be tested, add tests!
 - [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/blobtoolkit/tree/master/.github/CONTRIBUTING.md)- [ ] If necessary, also make a PR on the nf-core/blobtoolkit _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
 - [ ] Make sure your code lints (`nf-core lint`).
-- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
+- [ ] Ensure the unit test suite passes (`nextflow run . -profile test,singularity,sanger --outdir <OUTDIR>`).
+- [ ] Ensure the full test suite passes (`nextflow run . -profile test_full,singularity,sanger --outdir <OUTDIR>`).
```

Comment:
Add a note that the pass in the checks below only indicates that the job has been submitted, not that it has been completed successfully. The contributor needs to go to `farm5` and check that the job has indeed completed successfully before ticking this box.

Reply:
should check on the tower first using the link at the bottom of your ci job, this will be enough.
more details, check on the farm.